docker/Dockerfile.webapp

```diff
@@ -1,15 +1,20 @@
 FROM local/socorro_base
 
-# Install nodejs and npm
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y npm nodejs nodejs-legacy
+# Install nodejs and npm from Nodesource's 8.x branch
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y curl
+RUN DEBIAN_FRONTEND=noninteractive curl -sL https://deb.nodesource.com/setup_8.x | bash -
```

Comment:
I couldn't find a better way to get a recent node installed on Debian. Should we maybe copy this script into the repo instead of downloading and executing it?

Reply:
We should download it and put it in the repo so we know what we're executing.

Normandy is doing this:

https://github.com/mozilla/normandy/blob/master/recipe-server/Dockerfile#L9

That project is using python3.6-slim and we're using python2.7.13-slim (we should upgrade to python2.7.14-slim). So theoretically it should be in the same boat debian-wise. Can we reuse that?